README.md

```diff
@@ -14,12 +14,15 @@
 
 ## Live Demo
 
-[Live Demo Link](https://damdafayton.github.io/to-do-list)
+[Live Demo Link](https://damdafayton.github.io/to-do-list/dist)
 
 
 ## Getting Started
 
-Download `\dist` folder and click `index.html`.
+- To run: download `\dist` folder and click `index.html`.
+- To test :
+    - `npm install jest --global`
+    - `$ jest`
```

Comment:
- [ ] **[OPTIONAL]** I would recommend you add the correct command for actually running the tests ie `npm test`

Reply:
command is correct

Reply:
Oh yeah. You are right. Forgot to remove that